api/urls.py

```diff
@@ -44,4 +45,7 @@
 
     # Feedback
     path('add-feedback', feedback_views.add_feedback, name="add-feedback"),
+
+    # Currency Conversion
+    path('get-conversion-rate', currencyconverter_views.get_currency_exchange_rate, name="get-conversion-rate")
```

Comment:
have a more generic name like `currency_views` so that it can have other features in future as well